rosidl_generator_py/cmake/rosidl_generator_py_generate_interfaces.cmake

```diff
@@ -208,6 +215,8 @@ if(APPLE OR WIN32)
       "error code ${_result}")
   endif()
   target_include_directories(${_target_name_lib} PUBLIC "${_output}")
+
+  message(STATUS "Using NUMPY_INCLUDE_DIR: ${_output}")
```

Comment:
Nitpick: `NUMPY_INCLUDE_DIR` suggest it is a CMake constant and could be misleading. Proposing: `Using numpy include directory:` instead.

Also moving the `message()` call above the `target_include_directories()` call would be nice.

Reply:
Agreed, I'll make those changes.